src/qt/pivx/settings/settingsexportcsv.cpp

```diff
@@ -55,32 +51,38 @@ SettingsExportCSV::SettingsExportCSV(PIVXGUI* _window, QWidget *parent) :
     setFilterAddressBook(ui->comboBoxSortAddressType, lineEditAddressBook);
     ui->comboBoxSortAddressType->setCurrentIndex(0);
 
-    connect(ui->pushButtonSave, SIGNAL(clicked()), this, SLOT(exportClicked()));
     connect(ui->pushButtonDocuments, &QPushButton::clicked, [this](){selectFileOutput(true);});
+    connect(this, &SettingsExportCSV::fileSelectedTxes, [this](QString filename){exportTxes(filename);});
```

Comment:
No need for a lambda here, can change to
```cpp
connect(this, &SettingsExportCSV::fileSelectedTxes, this, &SettingsExportCSV::exportTxes);
```

Reply:
yes, will update this once #1049 is merged.
This will need a rebase anyway